Sources/Common/Util/Log.swift

```diff
@@ -89,12 +89,31 @@ public class ConsoleLogger: Logger {
     public func error(_ message: String) {
         printMessage("🛑 \(message)", .error)
     }
-    #else
-    // At this time, Linux cannot use `os.log` or `OSLog`. Instead, use: https://github.com/apple/swift-log/
-    // As we don't officially support Linux at this time, no need to add a dependency to the project.
-    // therefore, we are not logging if can't import os.log
-    public func debug(_ message: String) {}
-    public func info(_ message: String) {}
-    public func error(_ message: String) {}
-    #endif
+
+    public static func logMessageToConsole(_ message: String, level: CioLogLevel) {
+        #if canImport(os)
+        // Unified logging for Swift. https://www.avanderlee.com/workflow/oslog-unified-logging/
+        // This means we can view logs in xcode console + Console app.
+        if #available(iOS 14, *) {
+            let logger = os.Logger(subsystem: self.logSubsystem, category: self.logCategory)
+            logger.log(level: level.osLogLevel, "\(message, privacy: .public)")
+        } else {
+            let logger = OSLog(subsystem: logSubsystem, category: logCategory)
+            os_log("%{public}@", log: logger, type: level.osLogLevel, message)
+        }
+        #else
+        // At this time, Linux cannot use `os.log` or `OSLog`. Instead, use: https://github.com/apple/swift-log/
+        // As we don't officially support Linux at this time, no need to add a dependency to the project.
+        // therefore, we are not logging if can't import os.log
+        #endif
+    }
+}
+
+// Log messages to customers when the SDK is not initialized.
+// Great for alerts when the SDK may not be setup correctly.
+// Since the SDK is not initialized, dependencies graph is not created.
+// Therefore, this is just a function that's available at the top-level available to all
+// of the SDK without the use of dependencies.
+public func sdkNotInitializedAlert(_ message: String) {
```

Comment:
I like the idea but if all its doing is logging a message why are we calling it, `sdkNotInitializedAlert` because maybe we might have more alerts that could utilize this method?

Reply:
I thought about making the naming more generic. However, I found some cons with that approach. 

In the function body, you see `"⚠️ \(message)", level: .error`. So, we are prefixing all log messages with `⚠️ ` and we are using the log level of error. 

If we were to rename this function to something more generic:
```swift
public func logMessage(_ message: String, level: CioLogLevel) {
	ConsoleLogger.logMessageToConsole(message, level: level)
}
```

* Everywhere in the code that current calls `sdkNotInitializedAlert()` will now have copy/pasted code in it.

```swift
// before
sdkNotInitializedAlert("foo")

// after
logMessage("⚠️ foo", level: .error)
```

* There is a small number of edge cases where we need to log a message and the dependency injection graph is not yet constructed. Therefore, I don't believe at this time we will encounter a problem where the number of functions similar to `sdkNotInitializedAlert` grows. Currently, the only edge case I can consider needing logging without a DI graph is when the SDK is not initialized.

Reply:
I am leaning towards the current implementation of `sdkNotInitializedAlert` because
* A function name that's more generic such as `logMessage` doesn't tell the developer when to use that function to log a message and when to use the `Logger` that's inside of the DI graph. A function name such as `sdkNotInitializedAlert` suggests that this only be used when the SDK is not initialized. Otherwise, use the logger in the DI graph. 
* I like how DRY `sdkNotInitializedAlert`'s implementation is where the icon and log level are encapsulated into 1 place in the code. If we were to replace it with: `logMessage(message: "foo", level: .error, icon: .warning)`, it would no longer be DRY.